scenario_tests/collations/test.sh

```diff
@@ -8,17 +8,23 @@ source ../test-env.sh
 # Run service
 docker-compose up -d
 
+if [[ -n "${PRINT_TEST_LOGS}" ]]; then
+  docker-compose logs -f &
+fi
+
 sleep 30
 
 services=("pg" "pg-new")
 
 for service in "${services[@]}"; do
 
   # Execute tests
-  until docker-compose exec $service pg_isready; do
+  until docker-compose exec -T $service pg_isready; do
```

Comment:
All `docker-compose exec` must have `-T` flag so we can see the output in Github Action (it means no pseudo TTY allocation). But this is also harmless in local tests